crates/ruff_python_parser/src/lexer.rs

```diff
@@ -573,7 +573,7 @@ impl<'source> Lexer<'source> {
                     self.cursor.bump(); // '\'
                     if matches!(self.cursor.first(), '{' | '}') {
                         // Don't consume `{` or `}` as we want them to be emitted as tokens.
```

Comment:
Nit: Update the comment to explain that these are handled in the next iteration?